README.md

```diff
@@ -69,6 +69,8 @@ In the auto-generated `config/deploy.rb` file, add the following line to set the
 set :branch, ENV["BRANCH"] || "main"
 ```
 
+The auto-generated `config/deploy.rb` file also includes a write-version task that writes the branch name, commit SHA, and release date of the current deployment to `{yourwebsite}/version.txt`.
```

Comment:
We've switched the guidance for this project to using it as a reference instead of using it to generate an initial project. So I think we need language like, "Add the `write-version` task to give your new project a public page with the branch name, commit SHA, and release date of the current deployment."